tensorrt_llm/_torch/auto_deploy/models/patches/modelopt.py

```
@@ -0,0 +1,112 @@
+    """
+    Monkey-patch modelopt.torch.quantization.tensor_quant._tensor_quant so the
+    legacy CPU path in FakeTensorQuantFunction.forward uses our export-safe custom op.
+    """
+    try:
+        import modelopt.torch.quantization.tensor_quant as tq
+    except Exception:
+        # raise RuntimeError(
+        #     "Failed to import modelopt.torch.quantization.tensor_quant; "
+        #     "ensure ModelOpt is on PYTHONPATH."
+        # ) from e
+        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)